pkg/api/configsync/v1beta1/helmconfig.go

```diff
@@ -79,11 +85,12 @@ type HelmBase struct {
 	// +optional
 	IncludeCRDs bool `json:"includeCRDs,omitempty"`
 
-	// period is the time duration between consecutive syncs. Default: 15s.
+	// period is the time duration between consecutive syncs. Default: 1hour.
```

Comment:
Per offline discussion, `consecutive syncs` seems confusing. We can update the comment to  `refetching chart` specifically.

Reply:
Updated the crd description here to be more specific